### PR TITLE
Tests for lms-subjects

### PR DESCRIPTION
**Pull request by beabinside**
Added tests for /common/subjects/

### CODE REVIEW

File: back-end/src/common/tests/conftest.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# pylint: disable=unused-argument,redefined-outer-name,import-outside-toplevel,invalid-name,too-many-arguments,redefined-builtin
+import pytest
+
+from auth.models import User
+from common.models.subjects import Subject
+
+SUPERUSER_EMAIL = "superuserfortests@mail.com"
```
Comment (war-kolbus) [inline back-end/src/common/tests/conftest.py:7]: You should use only one conftest.py. Now there 2: tests/conftest.py and src/conftest.py. Pls merge this 2 files

File: back-end/src/common/tests/test_subjects.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+def unpack_subject(subject: Subject):
+    return {
+        "id": subject.id,
+        "title": subject.title,
+        "annotation": subject.annotation
+    }
+
+
+@pytest.mark.django_db
```
Comment (war-kolbus) [inline back-end/src/common/tests/test_subjects.py:13]: Move this func to common

File: back-end/src/common/tests/test_subjects.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    wrong = su_client.get("/api/lms/subjects")
+    assert wrong.status_code == 301
+
+    right = su_client.get("/api/lms/subjects/")
+    assert right.status_code == 200
+
+
+@pytest.mark.django_db
+def test_get_subject_by_id(su_client, get_new_subject_data, create_subject):
+    subj_data = get_new_subject_data(title="Строевая подготовка 2")
```
Comment (war-kolbus) [inline back-end/src/common/tests/test_subjects.py:24]: You dont need it

File: back-end/src/common/tests/test_subjects.py [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    subject = create_subject(**subj_data)
+    word = subject.title.split()[1]
+
+    subject_response_get = su_client.get(f"/api/lms/subjects/?search={word}")
+    assert subject_response_get.status_code == 200
+
+    subject_response_get = subject_response_get.json()
+    assert len(subject_response_get) >= 1
+    for subject in subject_response_get:
+        print(subject)
```
Comment (war-kolbus) [inline back-end/src/common/tests/test_subjects.py:64]: You should get rid of "print"

**Review by war-kolbus — CHANGES_REQUESTED** [review]
Not so bad 4/5
You should also test same with permissions

**Review by ShishckovA — APPROVED** [review]

**Pull request merged**